### PR TITLE
2016 reviewer mail

### PR DESCRIPTION
**Pull request by sarostru**
First draft of the reviewer mail, this mail is intended to go out to the whole list of reviewers and only those who respond that they are interested in reviewing this year will receive further more detailed communications.

I have tried an informal approach, let me know of any thoughts, in particular I was thinking if there needs to be additional information that I am missing or if the style needs to be a bit more formal.

### CODE REVIEW

**Comment by sbenthall** [comment]
this looks good to me. If Katy has no objections, please pull.

File: reviews/invite-reviewer.txt.in [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Hello {{ reviewer }}!
+
+The SciPy 2016 Conference on Python in Science is fast approaching, and so too
```
Comment (katyhuff) [inline reviews/invite-reviewer.txt.in]: It's typically called the "Scientific Computing with Python" conference.

File: reviews/invite-reviewer.txt.in [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+is the deadline for Proceedings submissions (May 16th) and consequently the
+start of the review process.
+
+We welcome you to be a member of the Proceedings review Committee this year.
```
Comment (katyhuff) [inline reviews/invite-reviewer.txt.in]: Proceedings review Committee should either be all capitalized or not at all.

File: reviews/invite-reviewer.txt.in [hunk 3]
```diff
@@ -0,0 +1,23 @@
+Hello {{ reviewer }}!
+
+The SciPy 2016 Conference on Python in Science is fast approaching, and so too
+is the deadline for Proceedings submissions (May 16th) and consequently the
```
Comment (katyhuff) [inline reviews/invite-reviewer.txt.in]: no need to capitalize proceedings here.

File: reviews/invite-reviewer.txt.in [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+start of the review process.
+
+We welcome you to be a member of the Proceedings review Committee this year.
+This should require a little of your time spread out over the review period
+(May 16th to July 4th) to read, comment, and ultimately decide on three
```
Comment (katyhuff) [inline reviews/invite-reviewer.txt.in]: Usually, reviews are assigned by the proceedings chairs. This reduces the burden on the reviewers time and helps to make sure an even(ish) number of reviews accross papers. You can certainly try it this way (asking the reviewers to assign themselves reviews) but I'm skeptical many will make the extra effort and I suspect you'll get lots of reviews on the "cool" papers and zero reviews on the less interesting papers.

Comment (katyhuff) [inline reviews/invite-reviewer.txt.in]: On second thought, maybe I'm misunderstanding the phrase "decide on" in this context.

Maybe you're just saying they'll decide on whether to accept the papers that they have been assigned. That's fine. 
 You might use the word "evaluate" instead. Or, you could replace "that line up with your area of expertise" with "that we assign you, according to your expertise"

File: reviews/invite-reviewer.txt.in [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+submissions that line up with your area of expertise.
+
+In the spirit of Open Source software, the SciPy conference uses a fully open
```
Comment (katyhuff) [inline reviews/invite-reviewer.txt.in]: does Open Source need to be capitalized?

File: reviews/invite-reviewer.txt.in [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+and collaborative review process to make proceedings acceptance decisions.  
+To learn more about this, you can read about it on the proceedings github page,
+https://github.com/scipy-conference/scipy_proceedings.   
+
+Please let us know if you would like to participate this year and we will
+include you in subsequent communications.  Also, feel free to pass this message
+along to anyone else who may be interested in reviewing.
```
Comment (katyhuff) [inline reviews/invite-reviewer.txt.in]: Do you really want _anyone_ else? Maybe you could ask for suggestions of reviewers. Say "Please don't hesitate to let us know of community members who we should consider inviting to review."

File: reviews/invite-reviewer.txt.in (hunk 4 above)
Comment (sarostru) [inline reviews/invite-reviewer.txt.in]: Thanks for highlighting this Katy, I meant the latter, but the way its phrased is confusing, I'll simplify it.

File: reviews/invite-reviewer.txt.in (hunk 6 above)
Comment (sarostru) [inline reviews/invite-reviewer.txt.in]: I like yours better, thanks for the idea.

**Pull request merged**